extensions-contrib/prometheus-emitter/src/main/java/org/apache/druid/emitter/prometheus/Metrics.java

```diff
@@ -49,6 +49,9 @@
   private final ObjectMapper mapper = new ObjectMapper();
   public static final Pattern PATTERN = Pattern.compile("[^a-zA-Z_:][^a-zA-Z0-9_:]*");
 
+  private static final String HOST_LABEL_NAME = "hostName";
+  private static final String SERVICE_LABEL_NAME = "druidService";
```

Comment:
For consistency with the other emitters, say `StatsDEmitter`,

```suggestion
  private static final String TAG_HOSTNAME = "host_name";
  private static final String TAG_SERVICE = "druid_service";
```